importer/harvester.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+# -*- coding: utf-8  -*-
+"""
+Download and process KMB data.
+
+Download xml metadata about KMB images, and preprocess
+it as json files.
+Requires a settings.json file containing an API key
+and a list of keywords. Generates one json file per
+keywords.
+"""
+import requests
+from xml.dom.minidom import parseString
+import time
+
+import batchupload.common as common
+from importer.kmb_massload import parser
+
+SETTINGS = "settings.json"
```

Comment:
including a `settings.json.example` file would be useful to show the structure expected in the file (it could use the test api key)